google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/DocumentReference.java

```diff
@@ -354,6 +355,15 @@ public T apply(List<T> results) {
     return extractFirst(firestore.getAll(this));
   }
 
+  /** Reads the document referenced by this DocumentReference. */
```

Comment:
Javadoc should be more descriptive. something like `fetches the document reference in the form of the stream inside `apiStreamObserver`.

Reply:
done